web/client/api/GeoStoreDAO.js

```diff
@@ -40,7 +40,7 @@ var Api = {
             this.addBaseUrl(parseOptions(options))).then(function(response) {return response.data; });
     },
     getResourcesByCategory: function(category, query, options) {
-        let q = query || "*";
+        let q = query && "*";
```

Comment:
This is not the required fix. We should remove the MS2 filter from configuration. This is asking for * for each configured filter.